lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1267,12 +1267,6 @@ if (!sourceList)
     void ScriptContext::SetIsClosed()
     {
         this->isClosed = true;
-#if ENABLE_NATIVE_CODEGEN
-        if (m_remoteScriptContextAddr)
-        {
-            JITManager::GetJITManager()->CloseScriptContext(m_remoteScriptContextAddr);
```

Comment:
Having thought more about this, I think we should have Close call still. With cleanup only, I think it may prevent us from short circuiting JIT on scripts that are closing (I think the JIT work item will keep the ScriptContext alive?)

However, Close should never actually free the ServerScriptContext (to prevent the races we see now).